esp/src/src-react/components/forms/landing-zone/BlobImportForm.tsx

```diff
@@ -123,31 +115,18 @@ export const BlobImportForm: React.FunctionComponent<BlobImportFormProps> = ({
         }
     }, [selection, reset]);
 
-    return <Modal
-        titleAriaId={titleId}
-        isOpen={showForm}
-        onDismiss={closeForm}
-        isBlocking={false}
-        containerClassName={componentStyles.container}
-        dragOptions={dragOptions}
-    >
-        <div className={componentStyles.header}>
-            <span id={titleId}>{`${nlsHPCC.Import} ${nlsHPCC.Blob}`}</span>
-            <IconButton
-                styles={FormStyles.iconButtonStyles}
-                iconProps={FormStyles.cancelIcon}
-                ariaLabel={nlsHPCC.CloseModal}
-                onClick={closeForm}
-            />
-        </div>
-        <div className={componentStyles.body}>
-            <Stack>
-                <Controller
-                    control={control} name="destGroup"
-                    render={({
-                        field: { onChange, name: fieldName, value },
-                        fieldState: { error }
-                    }) => <TargetGroupTextField
+    return <MessageBox title={nlsHPCC.Import} show={showForm} setShow={closeForm}
```

Comment:
Wasn't sure if this was the kind of thing you meant about looking at titles. They previously included the type of the spray, but seems fine that they can all just read Import too.

Reply:
Yes it was!